tests/playwright.config.ts

```diff
@@ -95,10 +96,11 @@ const config: PlaywrightTestConfig = {
   // outputDir: 'test-results/',
 
   /* Run your local dev server before starting the tests */
-  // webServer: {
-  //   command: process.env.DEVSERVER ? "npm run start" : "npm run start:built",
-  //   port: 3000,
-  // },
+  webServer: {
```

Comment:
Unless this is commented out, I'm getting errors trying to run `test:updateSnaps` - seems like pnpm is not recognized in the Docker container? @YousefED can you see if this is the case for you? Went over it on call with Nick and we couldn't find a solution, so would be at least good to know if it's just a problem on my side.